### ISSUE TITLE
Pinch and Swipe Issue

### ISSUE THREAD

**Issue by df-fady** [mattbryson/TouchSwipe-Jquery-Plugin#116]
Hi,

I have a problem using the 'swipe' and 'pinch' events on iPad.
If i use only the 'pinch' event it works great, but when i add the 'swipe' event pinching is no longer working.
i did test "Pinch_and_Swipe" demo but with no luck.

Thank you

Regards,

**Comment by jekcom** [comment]
Hi, I had the same issue and I solved it by changing the "else if" into just "if", since else if was never triggered, at those lines in the following condition:

```
// SWIPE GESTURES
if (didSwipe() || hasSwipes()) { //hasSwipes as status needs to fire even if swipe is invalid
    //Trigger the swipe events...
    ret = triggerHandlerForGesture(event, phase, SWIPE);
}

// PINCH GESTURES (if the above didn't cancel)
if ((didPinch() || hasPinches()) && ret !== false) {
    //Trigger the pinch events...
    ret = triggerHandlerForGesture(event, phase, PINCH);
}
```

Now it seems to work fine.

**Comment by mattbryson** [comment]
This is fixed in 1.6.8 59e13867707fb559799eca6c70ba073bc3d46a91
